tests/unit/AgmExtensionEvaluation.spec.ts

```diff
@@ -139,7 +139,7 @@ describe('ExtensionEvaluation', () => {
     const yellowEvalBox = wrapper.find('.message-box')
 
     expect(yellowEvalBox.exists()).toBe(true)
-    expect(yellowEvalBox.isVisible())
+    expect(yellowEvalBox.isVisible()).toBe(true)
```

Comment:
If the conditional is a `v-if` then check if it exists.

if the conditional is a `v-show` then check if it's visible.

You don't need both here :)

Reply:
Fixed, thanks.